### PR TITLE
docs: Add Shotstack showcase user

### PR DESCRIPTION
**Pull request by jeffski**
## Motivation

I would like to add Shotstack to the showcase of users. You can see our docs generated with Docusaurus here: https://shotstack.io/docs/guide/

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Run and check showcase page.

## Related PRs

None

### CODE REVIEW

**Comment by jeffski** [comment]
Signed the CLA just now for our business; Shotstack Pty Ltd.

**Comment by Josh-Cena** [comment]
Thanks

The screenshot shows the main site which doesn't seem Docusaurus-generated. Should we include the screenshot to the actual docs instead?

**Comment by jeffski** [comment]
> Thanks
> 
> The screenshot shows the main site which doesn't seem Docusaurus-generated. Should we include the screenshot to the actual docs instead?

Have updated to use the documentation home page instead.

**Comment by Josh-Cena** [comment]
Thanks, please check the build failure message & add the `opensource` tag

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1250,6 +1250,14 @@ const Users: User[] = [
     source: 'https://github.com/shabados/docs',
     tags: ['opensource', 'design'],
   },
+  {
+    title: 'Shotstack',
+    description: 'The cloud video editing API',
+    preview: require('./showcase/shotstack.png'),
+    website: 'https://shotstack.io/docs/guide/',
+    source: 'https://github.com/shotstack/documentation',
+    tags: ['product', 'design'],
```
Comment (slorber) [inline website/src/data/users.tsx:1259]:
```suggestion
    tags: ['product', 'design', 'opensource'],
```

**Comment by slorber** [comment]
Thanks 👍 

Wonder what are your thoughts about Remotion ;)

**Comment by jeffski** [comment]
Thanks @slorber. I have to admit Remotion is very good. A different approach to how our system works though.

**Pull request merged**